pyCRTM.py

```diff
@@ -200,7 +200,7 @@ def setupGases(self):
         max_abs = len(self.usedGases)
         nprof, nlay = self.profiles.T.shape 
         self.traceConc = np.zeros([nprof,nlay,max_abs])
-        self.traceIds = np.zeros(max_abs, dtype=np.int)
+        self.traceIds = np.zeros(max_abs, dtype=int)
```

Comment:
I think the new numpy `dtype` is `np.int_`, which should be compatible with Python `int`.
But standard python `int` is good, too.